### PR TITLE
Compose downloadable fonts

### PR DESCRIPTION
**Pull request by gtxtreme**
As per the Google recommend docs and the sample `JetCast` project, we could include fall back fonts, however without them this seems to work 

Having a fallback `.ttf` or `.otf` or any other format of font asset under `fonts/` may cause an increase in the app size and thus we don't want to do that 

References:

`JetCast` by Google : https://github.com/android/compose-samples/blob/main/Jetchat/app/src/main/java/com/example/compose/jetchat/theme/Typography.kt

`Official Docs` : https://developer.android.com/jetpack/compose/text#downloadable-fonts

### CODE REVIEW

**Comment by shounak-mulay** [comment]
@gtxtreme I think the fallback font should default to the system font we don't manually provide one.

File: resources/src/main/res/values/font_certs.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
...cm9pZDEQMA4GA1UECxMHQW5kcm9pZDEQMA4GA1UEAxMHQW5kcm9pZDEiMCAGCSqGSIb3DQEJARYTYW5kcm9pZEBhbmRyb2lkLmNvbTCCASAwDQYJKoZIhvcNAQEBBQADggENADCCAQgCggEBANbOLggKv+IxTdGNs8/TGFy0PTP6DHThvbbR24kT9ixcOd9W+EaBPWW+wPPKQmsHxajtWjmQwWfna8mZuSeJS48LIgAZlKkpFeVyxW0qMBujb8X8ETrWy550NaFtI6t9+u7hZeTfHwqNvacKhp1RbE6dBRGWynwMVX8XW8N1+UjFaq6GCJukT4qmpN2afb8sCjUigq0GuMwYXrFVee74bQgLHWGJwPmvmLHC69EH6kWr22ijx4OKXlSIx2xT1AsSHee70w5iDBiK4aph27yH3TxkXy9V89TDdexAcKk/cVHYNnDBapcavl7y0RiQ4biu8ymM8Ga/nmzhRKya6G0cGw8CAQOjgfwwgfkwHQYDVR0OBBYEFI0cxb6VTEM8YYY6FbBMvAPyT+CyMIHJBgNVHSMEgcEwgb6AFI0cxb6VTEM8YYY6FbBMvAPyT+CyoYGapIGXMIGUMQswCQYDVQQGEwJVUzETMBEGA1UECBMKQ2FsaWZvcm5pYTEWMBQGA1UEBxMNTW91bnRhaW4gVmlldzEQMA4GA1UEChMHQW5kcm9pZDEQMA4GA1UECxMHQW5kcm9pZDEQMA4GA1UEAxMHQW5kcm9pZDEiMCAGCSqGSIb3DQEJARYTYW5kcm9pZEBhbmRyb2lkLmNvbYIJANWFuGx90071MAwGA1UdEwQFMAMBAf8wDQYJKoZIhvcNAQEEBQADggEBABnTDPEF+3iSP0wNfdIjIz1AlnrPzgAIHVvXxunW7SBrDhEglQZBbKJEk5kT0mtKoOD1JMrSu1xuTKEBahWRbqHsXclaXjoBADb0kkjVEJu/Lh5hgYZnOjvlba8Ld7HCKePCVePoTJBdI4fvugnL8TsgK05aIskyY0hKI9L8KfqfGTl1lzOv2KoWD0KWwtAWPoGChZxmQ+nBli+gwYMzM1vAkP+aayLe0a1EQimlOalO762r0GXO0ks+UeXde2Z4e+8S/pf7pITEI/tP+MxJTALw9QUWEv9lKTk+jkbqxbsh8nfBUapfKqYn0eidpwq2AzVp3juYl7//fKnaPhJD9gs=
```
Comment (shounak-mulay) [inline resources/src/main/res/values/font_certs.xml:9]: Are these certs sensitive? Is it safe to put them in git?

**Review by shounak-mulay — APPROVED** [review]
Looks good.
Just one query about the certs.

Once the lint checks pass we are good to merge.

File: resources/src/main/res/values/font_certs.xml (hunk 1 above)
Comment (gtxtreme) [reply resources/src/main/res/values/font_certs.xml:9]: Yup if you see the package name and everything it belongs to Google and this cert file remains the same so it's a public certificate, everyone uses the same

**Pull request merged**